core/__tests__/tasks/profile/completeImport.ts

```diff
@@ -1,191 +0,0 @@
-import { helper } from "@grouparoo/spec-helper";
-import { api, config, task, specHelper } from "actionhero";
-import { Group, Profile, ProfileProperty, Property } from "../../../src";
-
-describe("tasks/profile:completeImport", () => {
-  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
-  beforeEach(async () => await api.resque.queue.connection.redis.flushdb());
-  beforeAll(async () => await helper.factories.properties());
-
-  let group: Group;
-
-  beforeAll(async () => {
-    group = await helper.factories.group();
-    await group.update({ type: "calculated" });
-    await group.setRules([
-      {
-        key: "lastName",
-        match: "Mario",
-        operation: {
-          op: config.sequelize.dialect === "postgres" ? "iLike" : "like",
-        },
-      },
-    ]);
-    await group.update({ state: "ready" });
-  });
-
-  describe("profile:completeImport", () => {
-    test("can be enqueued", async () => {
-      await task.enqueue("profile:completeImport", {
-        profileIds: ["abc123"],
-        toExport: true,
-      });
-      const found = await specHelper.findEnqueuedTasks(
-        "profile:completeImport"
-      );
-      expect(found.length).toEqual(1);
-    });
-
-    test("it will not export the profile if it is not ready", async () => {
-      const profile = await helper.factories.profile();
-      await profile.import();
-      await profile.update({ state: "pending" });
-
-      await specHelper.runTask("profile:completeImport", {
-        profileIds: [profile.id],
-        toExport: true,
-      });
-
-      await profile.reload();
-      expect(profile.state).toBe("pending");
-
-      const foundTasks = await specHelper.findEnqueuedTasks("profile:export");
-      expect(foundTasks.length).toBe(0);
-
-      await profile.destroy();
-    });
-
-    test("it will not export the profile if a profile property is not ready", async () => {
-      const profile = await helper.factories.profile();
-      await profile.import();
-      await profile.update({ state: "ready" });
-
-      const emailProperty = await Property.findOne({ where: { key: "email" } });
-      const profileProperty = await ProfileProperty.findOne({
-        where: { profileId: profile.id, propertyId: emailProperty.id },
-      });
-      await profileProperty.update({ state: "pending" });
-
-      await specHelper.runTask("profile:completeImport", {
-        profileIds: [profile.id],
-        toExport: true,
-      });
-
-      const foundTasks = await specHelper.findEnqueuedTasks("profile:export");
```

Comment:
I'd still suggest keeping these tests - moving them into `core/__tests__/tasks/profile/checkReady.ts`

Reply:
I don't think that it's possible to get a `ready` profile with a property that's `pending`, but the `completeImport` task was previously doing this check. I added it back to the `enqueueExports` task and moved the test over.